Common/Code.ts

```diff
@@ -24,7 +24,7 @@ interface IntegratedApp {
   getBody(): GoogleAppsScript.Document.Body | GoogleAppsScript.Slides.Slide[];
   getActive(): GoogleAppsScript.Document.Document | GoogleAppsScript.Slides.Presentation;
   getPageWidth(): number;
-  undoImage(delim: Delimiter): -1 | -2 | 1 | -4 | -3;
+  undoImage(delim: Delimiter): -1 | -2 | 1 | -4 | -5 | -3;
```

Comment:
It might be useful to change this return system to use enums or flags instead of this esoteric numbering system for the future :)